modyn/evaluator/internal/pytorch_evaluator.py

```diff
@@ -64,7 +64,6 @@ def _prepare_dataloader(
             dataset,
             batch_size=evaluation_info.batch_size,
             num_workers=evaluation_info.num_dataloaders,
-            timeout=60 if evaluation_info.num_dataloaders > 0 else 0,
```

Comment:
otherwise for criteo it times out

Reply:
we should not merge this. I don't understand why or how it would time out, need to investigate myself. We should not wait for over 60 seconds to receive data, even in the criteo case. Have not encountered this in my benchmarks so far!